sessionctx/variable/session.go

```diff
@@ -447,6 +447,9 @@ type SessionVars struct {
 	isolationReadEngines map[kv.StoreType]struct{}
 
 	PlannerSelectBlockAsName []ast.HintTable
+
+	// Lock wait timeout for pessimistic transaction in milliseconds, `innodb_lock_wait_timeout` is in seconds
```

Comment:
put `is in seconds` at the last is too easy to be misunderstood as `LockWaitTimeout` is in seconds.